src/wallet/users.spec.ts

```diff
@@ -378,14 +374,10 @@ describe("test user wallets", () => {
             version: 2,
         });
 
-        let serialized = transaction.serializeForSigning();
+        let serialized = transactionComputer.computeBytesForSigning(transaction);
         let signature = await signer.sign(serialized);
         let guardianSignature = await guardianSigner.sign(serialized);
 
-        assert.deepEqual(
-            serialized.toString(),
-            `{"nonce":0,"value":"0","receiver":"erd1cux02zersde0l7hhklzhywcxk4u9n4py5tdxyx7vrvhnza2r4gmq4vw35r","sender":"erd1l453hd0gt5gzdp7czpuall8ggt2dcv5zwmfdf3sd3lguxseux2fsmsgldz","gasPrice":1000000000,"gasLimit":50000,"data":"Zm9v","chainID":"1","version":2,"options":2,"guardian":"erd1spyavw0956vq68xj8y4tenjpq2wd5a9p2c6j8gsz7ztyrnpxrruqzu66jx"}`,
-        );
```

Comment:
I am not really sure why these kind of asserts have been deleted.

Reply:
before we where serializing the transaction before signing it and with this check we test the serialization now all this happens in the transaction computer and it's enough to just check the signature